diff --git a/node_modules/hexo-theme-redefine/layout/pages/post/toc.ejs b/node_modules/hexo-theme-redefine/layout/pages/post/toc.ejs
index 0d0d9df..2e4de39 100755
--- a/node_modules/hexo-theme-redefine/layout/pages/post/toc.ejs
+++ b/node_modules/hexo-theme-redefine/layout/pages/post/toc.ejs
@@ -3,7 +3,7 @@
 		<div class="toc-title"><%- __('toc') %></div>
 		<div class="page-title"><%= page.title %></div>
 		<%- toc(
-                page.content,
+                page.encrypt ? page.origin : page.content,
                 {
                     class: 'nav',
                     list_number: theme.articles.toc.number,
diff --git a/node_modules/hexo-theme-redefine/scripts/filters/encrypt.js b/node_modules/hexo-theme-redefine/scripts/filters/encrypt.js
index e4bcc20..36e3b07 100755
--- a/node_modules/hexo-theme-redefine/scripts/filters/encrypt.js
+++ b/node_modules/hexo-theme-redefine/scripts/filters/encrypt.js
@@ -125,7 +125,6 @@ hexo.extend.filter.register(
     data.content += `<link href="${hexo.config.root}css/hbe.style.css" rel="stylesheet" type="text/css"><script data-swup-reload-script type="module" src="${hexo.config.root}js/plugins/hbe.js"></script>
 <script data-swup-reload-script type="module">
 import {initHBE} from "${hexo.config.root}js/plugins/hbe.js";
-  log.info(ensurePrefix("hexo-blog-encrypt: loaded."));
     initHBE();
 </script>
 `;
